public/Sjofn/practice/sjofn4.html

```diff
@@ -0,0 +1,32 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Practice 4</title>
+</head>
+<body>
+<!--<button onclick="instruction1()">Open console and click me</button>-->
+</body>
+<script>
+    function getDatetime() {
+        console.log(new Date());
+    }
+    //getDatetime();
+    function getPowFive(z) {
+        console.log(Math.pow(z,5));
+    }
+    let z = prompt("z = ", " ");
```

Comment:
It is really cool that you've found a way to receive input via `prompt`, but, unfortunately, it wasn't required by the task. 
Please, remove all of the instructions, except the functions (according to the task's description).